packages/core/src/components/tabs/tabs.tsx

```diff
@@ -265,15 +342,52 @@ export class Tabs extends AbstractPureComponent<ITabsProps, ITabsState> {
 
         const tabIdSelector = `${TAB_SELECTOR}[data-tab-id="${this.state.selectedTabId}"]`;
         const selectedTabElement = this.tablistElement.querySelector(tabIdSelector) as HTMLElement;
+        const overflowElement = this.tablistElement.querySelector(OVERFLOW_SELECTOR) as HTMLElement;
+        const overflowChildren = overflowElement && overflowElement.children;
+
+        // Check if there are any overflow elements
+        let isOverflowing = false;
+        if (overflowChildren) {
+            for (const element in overflowChildren) {
+                if (
+                    overflowChildren.hasOwnProperty(element) &&
+                    overflowChildren[element].className === OVERFLOW_CONTENT_CLASS
```

Comment:
this is super brittle and (i believe) would break immediately if a user writes their own `overflowRenderer`. we need a better solution.

Reply:
Agreed, that's why I have a default overflowRenderer for the tabs that takes in the custom renderer a user wants to use from props. Everything renders out under the overflow content class which is just a class identifier that has no css. This happens regardless if they used the default or decided to write their own. This allows that loop to always find out if the overflowList has considered something to be overflowing and dropped it off to that renderer. I think this was the best approach based on what I saw in the codebase though you would know better. The other way I was thinking of grabbing that information is a public getter of some kind in the overflowList. Let me know if you can think of something else that's more appropriate for how y'all have architected things.